src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/ReleaseController.cs

```diff
@@ -275,11 +276,11 @@ public Task<ActionResult<ImportStatus>> GetDataUploadStatus(Guid releaseId, stri
                 .HandleFailuresOrNoContent();
         }
 
-        [HttpDelete("release/{releaseId}/chart/{subjectName}/{fileName}")]
+        [HttpDelete("release/{releaseId}/chart/{subjectName}/{id}")]
         public async Task<ActionResult> DeleteChartFile(
-            Guid releaseId, string subjectName, string fileName)
+            Guid releaseId, string subjectName, Guid id)
```

Comment:
Haven't noticed this before, but couldn't we use a `subjectId` here?

Reply:
Possibly - It's a bit annoying but the subject is stored against the meta detail for the uploaded file but given the import has take place then we might be able to do it - I'll raise another ticket for that though.

Reply:
On closer inspection this comes from the subject meta and there is no id - can investigate if there could be...

export interface TableDataSubjectMeta {
  publicationName: string;
  subjectName: string;

Reply:
done now